sdk/python/feast/infra/contrib/grpc_server.py

```diff
@@ -0,0 +1,55 @@
+from concurrent import futures
+
+import grpc
+import pandas as pd
+
+from feast.data_source import PushMode
+from feast.feature_store import FeatureStore
+from feast.protos.feast.serving.GrpcServer_pb2 import GrpcIngestFeatureResponse
+from feast.protos.feast.serving.GrpcServer_pb2_grpc import (
+    GrpcIngestFeatureServiceServicer,
+    add_GrpcIngestFeatureServiceServicer_to_server,
+)
+
+
+class GrpcIngestFeatureService(GrpcIngestFeatureServiceServicer):
+    fs: FeatureStore
+    sfv: str
+    to: PushMode
+
+    def __init__(self, fs, sfv, to):
+        self.fs = fs
+        self.sfv = sfv
+        self.to = to
+        super().__init__()
+
+    def GrpcIngestFeature(self, request, context):
+        features = {}
+        for i in request.features.keys():
+            features[i] = [request.features.get(i)]
+        rows = pd.DataFrame.from_dict(features)
+        if self.to == PushMode.ONLINE or self.to == PushMode.ONLINE_AND_OFFLINE:
+            self.fs.write_to_online_store(self.sfv, rows)
+        if self.to == PushMode.OFFLINE or self.to == PushMode.ONLINE_AND_OFFLINE:
+            self.fs.write_to_offline_store(self.sfv, rows)
+        return GrpcIngestFeatureResponse(status=True)
+
+
+class GrpcIngestFeatureServer:
+    def __init__(self, address, fs: FeatureStore, sfv_name, to, max_workers):
```

Comment:
why limit to just one sfv? seems better to keep it flexible?

Reply:
makes sense. added feature_view name to proto. ✅